src/promptflow-core/promptflow/_core/tool_meta_generator.py

```diff
@@ -261,10 +261,22 @@ def load_python_module_from_file(src_file: Path):
     try:
         spec.loader.exec_module(m)
     except Exception as e:
+        message = "Failed to load python module from file '{src_file}': {error_type_and_message}"
+        if (
+            isinstance(e, ImportError)
+            and hasattr(e, "msg")
+            and e.msg == "attempted relative import with no known parent package"  # pylint: disable=no-member
+        ):
+            message = (
+                "Failed to load python module from file '{src_file}' due to relative import."
+                + " Relative imports fail in loaded code that is not a known python module."
+                + " Please use absolute imports."
```

Comment:
Is there other cases that raise `ImportError` (i.e. a misspelling in the package name) here, though it is an absolute import? Is this error message always accurate for all the cases?

Reply:
Good point, there could be other sources for import errors (like the user just having a bug in their own save/loaded eval. I'll change the wording to be speculative.